utils/test_utils/test_utils.go

```diff
@@ -0,0 +1,54 @@
+package test_utils
+
+import (
+	"fmt"
+	"github.com/joshprzybyszewski/sudoku_fun/twodee_bitwise/common"
+	"github.com/joshprzybyszewski/sudoku_fun/utils/constants"
+)
+
+func SmartPuzzlesAreEqual(exp, act *common.SmartPuzzle) bool {
+	if exp.NumPlaced != act.NumPlaced {
+		println(fmt.Sprintf("NumPlaced expected: %v, actual: %v", exp.NumPlaced, act.NumPlaced))
+		return false
+	}
+
+	//if exp.Solver == act.Solver {
```

Comment:
I JUST WANT TO COMPARE MEMORY ADDRESSES OF THIS FUNCTION! WHY IS THAT SO HARD???????